protostar-rust/run_tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+pushd cairo-tests
+
+if [ $# -gt 0 ]; then
+  pushd "$1"
+  cargo run
+  popd # $1
+  exit 0;
+fi
+
+for DIR in */ ; do
+  pushd "$DIR"
+  cargo run
+  popd # $DIR
+done
+
+popd # cairo-tests
```

Comment:
We shouldn't have to use scripts like this. See e2e tests in #2033. This should be rebased.